src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java

```diff
@@ -267,6 +267,12 @@ public class MiscConfig {
     @FeatureToggle
     public boolean replaceRomanNumerals = false;
 
+    @Expose
+    @ConfigOption(name = "Thunder Bottle", desc = "Show a notification when you charge a Thunder Bottle.")
```

Comment:
your description is a bit missleading, not sure if mine is better though
the title could maybe also be more descriptive idk
```suggestion
    @ConfigOption(name = "Thunder Bottle", desc = "Show a notification when your Thunder Bottle is fully charged.")
```